src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.subsystems.hood;
+
+import edu.wpi.first.math.Pair;
+
+public class HoodConstants {
+    public static Pair<Integer, Integer> mechanism2dPose = new Pair<>(1, 1);
```

Comment:
Why not use `Pose2d`?

Reply:
5c91961

Reply:
This commit doesn't have any connection to the comment.

Reply:
3390514